soh/soh/Enhancements/randomizer/logic.cpp

```diff
@@ -1694,6 +1694,9 @@ namespace Rando {
                 if (BottleRandomizerGetToItemID.contains(randoGet)) {
                     itemId = BottleRandomizerGetToItemID[randoGet];
                 }
+                if (randoGet == RG_BOTTLE_WITH_BIG_POE && !ctx->GetOption(RSK_BIG_POE_COUNT).Get()) {
```

Comment:
I would prefer an explicit numeral comparison to 1, instead of a converting to boolean.

Reply:
The comparison would be `== 0` or ` + 1 == 1` since the option's 0 values is 1 big poe

Reply:
I went ahead & pushed as a separate commit handling multiple big poe bottles, if that's preferable it renders this obsolete